PROTOCOL.md

```diff
@@ -253,6 +253,47 @@ as of now. The add and remove file actions are stored as their individual column
 
 These files reside in the `_delta_log/_sidecars` directory.
 
+### Log Compaction Files
+
+Log compaction files resides in the `_delta_log` directory. A log compaction file from a start version `x` to an end version `y` will have the following name: 
```

Comment:
There isn't really any mention of writers here, tho much of it can probably be inferred.
Seems like we could be more clear by saying:
* Writers
  * Can optionally produce log compactions (with advice about when they might want to do so)
  * Compactions produced by applying action reconciliation to the commits to be compacted
* Readers
  * Can optionally consume log compactions, if available
  * The compaction replaces the corresponding commits during action reconciliation

Also probably worth pointing out that this is _not_ a table feature, because both writers and readers can safely access the table even if they know nothing about log compactions?

Reply:
Added Readers section.
Added Writer section except this point: `Compactions produced by applying action reconciliation to the commits to be compacted`. Isn't this the definition of log compaction which we have already explained in the initial part of this section?

Also explicitly pointing out that minor-compaction is not a table feature seems little out of context from PROTOCOL's perspective?